packages/node_modules/nav-frontend-alertstriper-style/src/index.less

```diff
@@ -13,6 +13,10 @@
   }
 
   &--advarsel {
+    .alertstripe__ikon {
+      border-right: 1px solid @redError!important;

```

Comment:
Stemmer det at streken til høyre for ikonet her faktisk skal være rød, når den i alle andre tilfeller er grå og ikke matcher med fargen på ikonet (@sergiohaisch)? Ser det er sånn i Zeplin, men dette ser ut som en feil, enten den ene eller andre veien. Fjern dette, så retter vi heller opp senere om det skal være sånn.

Reply:
Nei, strekken skal vare grå som alle de andre. Det er en feil i Zeplin